testV5/eth2DaiReserve.js

```diff
@@ -1436,6 +1436,555 @@ contract('Eth2DaiReserve', function(accounts) {
             assert.equal(expectedUserETHBal.valueOf(), newUserEthBal.valueOf(), "eth balance should be correct after traded, loop: " + i);
         }
     });
+
+    it("Should test set internal inventory failed sender is not admin", async function() {
+        try {
+            await reserve.setInternalInventoryData(
+                myDaiToken.address,
+                true,
+                0,
+                0,
+                0,
+                0,
+                {from: operator}
+            )
+            assert(false, "throw was expected in line above.")
+        } catch (e) {
+            assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+        }
+    });
+
+    it("Should test set internal inventory failed token is not listed", async function() {
+        let token = await TestToken.new("test token", "tst", 18);
+        try {
+            await reserve.setInternalInventoryData(
+                token.address,
+                true,
+                0,
+                0,
+                0,
+                0,
+                {from: admin}
+            )
+            assert(false, "throw was expected in line above.")
+        } catch (e) {
+            assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+        }
+    });
+
+    it("Should test set internal inventory failed data overflows", async function() {
+        let power32 = (new BigNumber(2)).pow(32);
```

Comment:
better call this out of range.
not overflow.

Reply:
Ok I change to out of range